src/views/layouts/_master.njk

```diff
@@ -6,8 +6,14 @@
     {% set page = {} %}
 {% endif %}
 
+
 {% if example %}
     {% set bodyClasses = bodyClasses + " patternlib-page--example" %}
+    {% if page.theme %}
```

Comment:
What's this for?

Reply:
So we can add a theme to an individual example. Look at the deployment.

Reply:
Ah cool I see what you've done here, I was an idiot and thought this was the main page-template.njk